integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -47,6 +47,21 @@ class SapSuccessFactorsContentMetadataExporter(ContentMetadataExporter):
         'price': 'price',
     }
 
+    def _apply_delete_transformation(self, metadata):
+        """
+        Specific transformations required for "deleting" a course on a SAP external service.
+        """
+        # Applying the metadata payload update to "delete" the course on SAP instances
+        metadata['status'] = 'INACTIVE'
+
+        # Sanity check as we've seen issues with schedule structure
+        metadata_schedule = metadata.get('schedule')
+        if metadata_schedule:
+            schedule = metadata_schedule[0]
+            if not schedule.get('startDate') or not schedule.get('endDate'):
```

Comment:
question: Do we care about the `active` property whether it's `true` or `false`?

Reply:
I don't believe so, the only comment I saw from @bcitro's original work was on start and end date being empty. But also maybe good to double check with him?

Reply:
start and end are the only ones I've seen errors for, so I think that's sufficient to check here